source/includes/kubernetes/_k8_configmaps.md

```diff
@@ -129,6 +129,61 @@ Return value:
 | `taskId` <br/>_string_     | The id corresponding to the create config maps task. |
 | `taskStatus` <br/>_string_ | The status of the operation.                         |
 
+<!-------------------- REPLACE A CONFIG MAP -------------------->
+
+#### Replace a config map
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/configmaps/configmaps-name/default"
+  Content-Type: application/json
+  {
+  "apiVersion": "v1",
+  "kind": "ConfigMap",
+  "metadata": {
+    "name": "game-demo"
```

Comment:
The name here doesn't match the one in the URL. It could potentially cause confusion with API users.

Reply:
🙈 29bd284da5bdcce53262efad30eeba8c620e8c97